packages/tools/src/commands/create-plugin/hooks.ts

```diff
@@ -2,12 +2,45 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2019-Present Datadog, Inc.
 
-import { dim } from '@dd/tools/helpers';
+import { dim, green, grey, yellow } from '@dd/tools/helpers';
 import { outdent } from 'outdent';
 
-import type { AllHookList, AnyHook, BundlerHook, Hook, UniversalHook } from './types';
+import type {
+    AllHookList,
+    AnyHook,
+    BundlerHook,
+    Choice,
+    TypeOfPlugin,
+    UniversalHook,
+} from './types';
 
-export const bundlerHooks: Record<BundlerHook, Hook> = {
+export const pluginTypes: Record<TypeOfPlugin, Choice> = {
+    universal: {
+        name: `[${green('Recommended')}] Universal Plugin`,
+        descriptions: [
+            'Create a customer facing plugin.',
+            'A single implementation for all bundlers.',
+            'It will use hooks that are supported by all bundlers.',
+        ],
+    },
+    bundler: {
+        name: `[${yellow('Discouraged')}] Bundler Specific Plugin`,
+        descriptions: [
+            'Create a plugin that will be customer facing.',
```

Comment:
**Super nit:** it feels weird to me to have different wording for the same thing ('Create a customer facing plugin.' / 'Create a plugin that will be customer facing.')

Reply:
Agreed 100%, I'll update.